lesson_25/homework_25.py

```diff
@@ -1,8 +1,61 @@
-import requests
 """
 Написати 25 XPath та 25 CSS локаторів для сайту https://qauto2.forstudy.space/
 """
 
 username = "guest"
 passwd = "welcome2qauto"
 url = f"https://{username}:{passwd}@qauto2.forstudy.space"
+
+#XPath
+HOME_MENU_BUTTON_XPATH = "//a[text()="Home"]"
```

Comment:
```suggestion
HOME_MENU_BUTTON_XPATH = "//a[text()='Home']"
```
У вас дужки неправильно розставлені всюди